avocado/core/messages.py

```diff
@@ -70,7 +70,8 @@ def __init__(self):
         self._handlers = {'log': [LogMessageHandler()],
```

Comment:
The commit says File_Message, but this is actually FileMessage, right?

Reply:
Yes, my bad